packages/editor/src/client/locales/en.json

```diff
@@ -788,7 +790,9 @@
         "subEdit": "Edit",
         "subTitle": "Subscription",
         "Unknown": "Unknown",
-        "userRoles": "User Roles"
+        "userRoles": "User Roles",
+        "authors": "Authors",
```

Comment:
If a user can have only one author, it might make sense to have it singular, so it's clear in the panel you can add only one author.